spec/integration/output_stream_spec.rb

```diff
@@ -17,7 +17,28 @@
             true
           end
         end
-      "
+      SPEC
```

Comment:
Why the change to the heredoc?  We haven't done that for our other integration specs.